### DISCUSSION TITLE
Greenland total mass balance from 1840 to next week

### DISCUSSION THREAD

**Discussion by PennyHow** [mdtanker/antarctic_plots#219] [Dataset requests]
- DOI: https://doi.org/10.22008/FK2/OHI23Z
- Daily updated data product: https://thredds.geus.dk/thredds/catalog/MassBalance/catalog.html
- Fetching through the thredds link should be fairly straightforward

**Comment by mdtanker** [comment]
Great! Thanks for suggesting this :) I'll try and add it in the next few weeks.
